packages/storybook/stories/ovhcloud-design-system/design-and-style/tailwind.mdx

```diff
@@ -0,0 +1,160 @@
+import { Meta, Source } from '@storybook/blocks';
+import { ExternalLink } from '../../../src/components/externalLink/ExternalLink';
+import { Heading } from '../../../src/components/heading/Heading';
+import { StorybookLink } from '../../../src/components/storybookLink/StorybookLink';
+import { HOME_TITLE } from '../../../src/constants/meta';
+
+<Meta title="OVHcloud Design System/Design and Style/Tailwind CSS Integration" />
+
+<Heading label="Tailwind CSS Integration" level={ 1 } />
+
+This guide covers how to integrate Tailwind CSS with the OVHcloud Design System (ODS) for both v3 and v4 versions.
+Tailwind CSS is a utility-first CSS framework that can be integrated with ODS components and design tokens, but it is not mandatory as ODS is not based on Tailwind.
+Technically, you don't need any configuration to make ODS work with Tailwind. These guides are here to add some auto-completion and best practices for Tailwind, but you can directly use the design tokens right out of the box without any additional configuration.
```

Comment:
Maybe we should isolate this in an `information` `<Message>` so that it stands out more?